sweep/sweeper.go

```diff
@@ -531,15 +586,23 @@ func (s *UtxoSweeper) signalAndRemove(outpoint *wire.OutPoint, result Result) {
 
 	// Inputs are no longer pending after result has been sent.
 	delete(s.pendingInputs, *outpoint)
+	delete(s.inputsByFeePref[pendInput.feePreference], *outpoint)
```

Comment:
Is the map itself for a particular fee pref ever deleted? Not really important though.

Reply:
No, but it is now.